webclient/index.html

```diff
@@ -681,6 +685,35 @@
         link.click();
     }
 
+
+
+    const configureAuthClient = async ()=>{
+
+        const config = {domain: "{{ .Env.AUTH0_DOMAIN }}", clientId: "{{ .Env.AUTH0_CLIENT_ID }}"};
+
+        auth0Client =  await auth0.createAuth0Client({
+            domain: config.domain,
+            clientId: config.clientId,
+            cacheLocation: 'localstorage'
+        });
+    }
+
+    const authentication = async () => {
+        await configureAuthClient();
```

Comment:
**(issue)** Please move client initialization to happen upon startup of the app to avoid unnecessary invocations.